source/javascripts/core/api/SecretApi.ts

```diff
@@ -164,28 +137,20 @@ async function getSecretValue({
 }: {
   appSlug: string;
   secretKey: string;
-  useApi?: boolean;
   signal?: AbortSignal;
 }): Promise<string | undefined> {
   if (RuntimeUtils.isWebsiteMode()) {
-    if (params.useApi) {
-      const response = await Client.get<SecretApiValueResponse>(getSecretValueFromApiPath(params.appSlug), {
-        signal,
-      });
-      return response.value;
-    }
-
     const response = await Client.get<SecretsMonolithResponse[number]>(getSecretItemPath(params), {
       signal,
     });
     return response.value;
   }
 
   // CLI mode
-  return Promise.reject(new Error('Getting secret environment value is only available in website mode'));
+  return '';
```

Comment:
Why don't we need the error message anymore?

Reply:
Because the CLI secrets response already has values.